### PR TITLE
EntityBuilder UpdateColumn to dynamic type of value

### PR DESCRIPTION
**Pull request by tomsync**
String value doesn't work for bool type on PgSQL

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/oqtane/oqtane.framework?pullRequest=1603) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/oqtane/oqtane.framework?pullRequest=1603) before we can accept your contribution.<br/><br/>:x: tomsync [sign now](https://cla.dotnetfoundation.org/oqtane/oqtane.framework?pullRequest=1603)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/oqtane/oqtane.framework?pullRequest=1603) it.</sub>

**Comment by sbwalker** [comment]
@tomsync since this API is used by external modules, this appears to potentially be a breaking change as the parameter type is being changed. Do you know if changing a type from string to dynamic causes compatibility issues?

**Comment by sbwalker** [comment]
I need to back out this PR as it is causing an error for other databases.

File: Oqtane.Server/Migrations/Tenant/02020001_AddPageIsClickable.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
             var pageEntityBuilder = new PageEntityBuilder(migrationBuilder, ActiveDatabase);
 
             pageEntityBuilder.AddBooleanColumn("IsClickable", true);
-            pageEntityBuilder.UpdateColumn("IsClickable", "1");
+            pageEntityBuilder.UpdateColumn("IsClickable", true);
```
Comment (hishamco) [inline Oqtane.Server/Migrations/Tenant/02020001_AddPageIsClickable.cs:22]: I'm not sure why you update the column while it's added as a boolean 'true'?!!

**Pull request merged**